Mathlib/Data/Sym/Card.lean

```diff
@@ -189,9 +189,26 @@ theorem card_subtype_not_diag [Fintype α] :
   exact and_iff_right ⟨a, mem_univ _, ha⟩
 #align sym2.card_subtype_not_diag Sym2.card_subtype_not_diag
 
-/-- Type **stars and bars** for the case `n = 2`. -/
+/-- **Stars and bars** for the case `n = 2`. -/
```

Comment:
Why? This says this is the version for fintypes, as opposed to the version for finsets